src/app.js

```diff
@@ -0,0 +1,36 @@
+go.app = function () {
+  var vumigo = require('vumigo_v02');
+  var App = vumigo.App;
+  var EndState = vumigo.states.EndState;
+  var utils = require('./utils');
```

Comment:
You won't be able to require `utils` this way, because for production everything is rolled into one file, `go-app.js`. Instead, where you use `utils`, just use `go.utils` instead and it will find it.